utils/browser_history.jsx

```diff
@@ -43,3 +43,12 @@ export const browserHistory = {
         }
     },
 };
+
+/**
+ * Returns the current history object.
+ *
+ * If you're calling this from within a React component, consider using the useHistory hook from react-router-dom.
+ */
+export function getHistory() {
```

Comment:
should naming a bit more descriptive makes sense here although we do have the comment above, something along lines
`getDesktopCompatibleHistory()`
`getHistory__DESKTOP_COMPATIBLE()`

Reply:
I don't think it needs a special name since the idea is that this would just get used everywhere someone's accessing the history that `useHistory` isn't available from. Most of the time, people using it shouldn't care that this history object is desktop-compatible or not